src/main.ts

```diff
@@ -16,6 +18,8 @@ async function bootstrap() {
     }),
   );
 
+  sgMail.setApiKey(sendgridConfig.apiKey);
```

Comment:
you should create a separate Sendgrid service, where send email is implemented and api key set